packages/core/src/_common/base/common_resources.js

```diff
@@ -0,0 +1,17 @@
+const BinarySocket = require('./socket_base');
+
+const TradingTimesBase = (() => {
```

Comment:
best to update the file name to `common_resource_base` or `trading_times_base`, in that way it is more descriptive.